### DISCUSSION TITLE
members ? in self hosted version?

### DISCUSSION THREAD

**Discussion by zakidze** [tolgee/tolgee-platform#2262] [Q&A] [answered]
i tryed installing self hosted version but i dont see the members tabs same as the website 

i see here there are 10 seats  how to invite people colaboration and  even in local lan???? with their own accounts
https://tolgee.io/pricing/self-hosted#pricing-toggle

**Comment by zakidze** [comment]
https://github.com/tolgee/tolgee-platform/discussions/1730

**Comment by JanCizmar** [comment, marked as answer]
Hey! You are looking for this configuration option: https://tolgee.io/platform/self_hosting/configuration#enabled

By default, auth is disabled for docker.

**Reply by JanCizmar** [reply]
Truth is, that the docs should contain this in the example docs. I've updated it.

https://github.com/tolgee/documentation/commit/c9e3307ea531b41d8c150e18c81a993743859bf8
